packages/panels/resources/views/components/header/index.blade.php

```diff
@@ -1,10 +1,14 @@
 @props([
     'actions' => [],
     'breadcrumbs' => [],
-    'heading',
-    'subheading' => null,
 ])
 
+@php
+    $heading = $attributes->get('heading');
```

Comment:
```suggestion
    // These are passed through in the bag otherwise Laravel converts View objects to strings prematurely.
    $heading = $attributes->get('heading');
```